2 table.sql

```diff
@@ -0,0 +1,106 @@
+/*
+Flight number, Departure Airport, Country, Arrival Airport, Country, Time Departing, 
+Time Arriving, Name, DOB (dd/mm/yyyy), Address, (Passport number, Issue date, Expiry date, Nationality) if provided
+
+
+
+use airlineDB
+go 
+drop table if exists dbo.Passenger
+go 
+create table dbo.Passenger(
+	PassengerID int not null identity primary key,
+	FlightNumber char(6) constraint flightnumber must be FLY[0-9][0-9][1-9]
+	DepartureAirport char(3) not null must be [a-z][a-z][a-z]
+	DepartureCountry varchar(30) not null not blank
+	ArrivalAirport char(3) not null must be [a-z][a-z][a-z]
+	ArrivalCountry varchar(30) not null not blank
+	TimeDeparting can't be more than one year past current date
+	TimeArriving 
+	PassengerFirstName varchar(30) not null not blank
+	PassengerLastName varchar(30) not null not blank
+	PassengerDateOfBirth date not null 16-90
+	PassengerAddress varchar(150) not null not blank
+	PassportNumber char(9) not null default " must be  [1-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9][0-9] unique- test if could enter null
+	PassportIssueDate date not null default " when international and to passports country, can't be more than 9 yrs and 6 mo old 
+		when age < 16, can't be more than 5 years old
+	PassportExpiryDate date not null default " 
+	Nationality not null default "
+	multi- column constraints
+		name, dob, address and flight # must be unique
+		time arriving must be greater than time departing
+		passport info must be all entered or all blank
+)
+
+*/
+
+use airlineDB
+go 
+drop table if exists dbo.Passenger
+go 
+create table dbo.Passenger(
+	PassengerID int not null identity primary key,
+	FlightNumber  char(6) not null 
+		constraint ck_Passenger_FlightNumber_must_be_FLY_followed_by_3_numbers check (FlightNumber like 'FLY[0-9][0-9][1-9]'),
```

Comment:
You are not allowing for a flight numbered 100. You blocked any flight ending with 0, will only flight 000 should be blocked.

Reply:
fixed